test/TestApp.svelte

```diff
@@ -87,7 +87,7 @@ th {
   <tbody>
     {#each $tests as test (test.name) }
       <tr>
-        <td>{ test.name }</td>
+        <td class="{test.result || ''}">{ test.name }</td>
```

Comment:
Is there a way to use Svelte's `class:` notation here?

Reply:
Hmm... I think it would look like this if we did:
```svelte
    {#each $tests as { name, expected, result } (test.name) }
      <tr>
        <td class="{result || ''}">{ name }</td>
        <td>{ JSON.stringify(expected) }</td>
        <td class="uppercase {test.result}">{ result || '' }</td>
        <td>{ details || '' }</td>
      </tr>
    {/each}
```